integrationtest/src/main/scala/io/gearpump/integrationtest/storm/StormClient.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gearpump.integrationtest.storm
+
+
+import backtype.storm.utils.DRPCClient
+import io.gearpump.integrationtest.Constants._
+import io.gearpump.integrationtest.{Shell, Docker}
+import io.gearpump.integrationtest.minicluster.{CommandLineClient, MiniCluster}
+
+class StormClient(cluster: MiniCluster, commandLineClient: CommandLineClient) {
+
+  private val STORM_HOST = "storm0"
+  private val STORM_CMD = "/opt/start storm"
+  private val STORM_STARTER_JAR = "/opt/gearpump/lib/storm/storm-starter-0.9.5.jar"
+  private val STORM_DRPC = "storm-drpc"
+  private val IMAGE = "stanleyxu2005/gpct-jdk8"
+  private val CONFIG_FILE = "storm.yaml"
+  private var drpcServerIp: String = null
+
+  def start(): Unit = {
+    Docker.createAndStartContainer(STORM_HOST,
```

Comment:
What is the propose of creating a storm client? You can expose more ports in MiniCluster's constructor. Do you only want to obtain special ip when cluster is getting started?